apps/docs/src/app/platform/component-docs/platform-object-attribute/platform-object-attribute-docs.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import * as objectAttribute from '!raw-loader!./platform-object-attribute-examples/platform-object-attribute-example.component.html';
+import * as objectAttributeTruncate from '!raw-loader!./platform-object-attribute-examples/platform-object-attribute-truncate-example.component.html';
+import { ExampleFile } from '../../../documentation/core-helpers/code-example/example-file';
```

Comment:
Suggestion:
Maybe it will be better to split outer imports (like angular, rxjs and etc.) with inner imports?
With one empty line-separator